ui/pages/unlock-page/index.scss

```diff
@@ -25,7 +25,7 @@
     &__beta {
       /* these colors should be used on both light and dark mode */
       background: var(--brand-colors-blue-blue500);
-      color: var(--brand-colors-white-white000);
+      color: var(--brand-colors-white);
       padding: 3px 6px;
```

Comment:
### After
Beta text remains white

<img width="361" alt="Screenshot 2024-06-10 at 3 26 13 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/a431e360-bfc0-42c5-96d9-91a52ffd3c4e">

Reply:
should this be using a themed token instead?

Reply:
No, see comment on line 26 `/* these colors should be used on both light and dark mode */`

Reply:
Good question! I think originally it used brand colors here because the designer wanted the text to remain white but it would look strange once we update the colors so I'll update to theme. Updated here https://github.com/MetaMask/metamask-extension/pull/24953/commits/ddaeba1ec3fbbbf6e4895f09c4ac0864f7d14d9f

<img width="366" alt="Screenshot 2024-06-10 at 4 28 09 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/6cf454ac-629e-4288-bb69-7a07ed739553">
<img width="361" alt="Screenshot 2024-06-10 at 4 28 35 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/6c8ca914-f887-43ac-b2a2-19b771ec5e0b">

Reply:
nice!